src/NServiceBus.Core/Persistence/PersistenceExtensions.cs

```diff
@@ -11,7 +11,7 @@
     /// This class provides implementers of persisters with an extension mechanism for custom settings for specific storage
     /// type via extension methods.
     /// </summary>
-    /// <typeparam name="T">The persister definition eg <see cref="NServiceBus.InMemory" />, <see cref="MsmqTransport" />, etc.</typeparam>
+    /// <typeparam name="T">The persister definition eg <see cref="NServiceBus.InMemory" />, etc.</typeparam>
```

Comment:
Need to double check but InMemory in a cref seams strange since the actual class name is InMemoryPersistence